### PR TITLE
0.6.0 changelog

### PR DESCRIPTION
**Pull request by arnikola**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
 =========
+
+# 0.6.0 (2019-02-19)
+
+## New Features
+
+- **M3DB** (Config): Simplify M3 config options (#1371)
+- **M3Coordinator**: Improvements to database creation API (#1350)
+- **M3Coordinator**: Enforce ID generation scheme explicitly added to configs (#1381)
+- **M3Query**: Add quantile_over_time (#1367) and histogram_quantile (#1372) functions
```
Comment (benraskin92) [inline CHANGELOG.md]: `functions` => `Prometheus functions`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+- **Documentation**: Additional documentation for namespace setup and configuration (#1350), etcd (#1354), and M3Coordinator ID generations schemes (#1381, #1385)
+
+## Performance
+
+- **M3DB** (Index): Add posting list cache (#1370)
```
Comment (richardartoul) [inline CHANGELOG.md]: Can we add something like: This should result in a massive M3DB query performance for most workloads.

File: src/query/storage/index.go [hunk 3]
```diff
@@ -90,17 +91,50 @@ func FetchOptionsToM3Options(fetchOptions *FetchOptions, fetchQuery *FetchQuery)
 	}
 }
 
+var (
```
Comment (richardartoul) [inline src/query/storage/index.go]: should these changes be in this P.R?

Comment (arnikola) [reply src/query/storage/index.go]: Oops, not sure how they got included :\

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+- **Documentation**: Additional documentation for namespace setup and configuration (#1350), etcd (#1354), and M3Coordinator ID generations schemes (#1381, #1385)
+
+## Performance
+
+- **M3DB** (Index): Add posting list cache (#1370) that should result in a massive improvement for M3DB query performance for most workloads
+
+## Bug fixes
+
+- **M3DB** (Index): Fix to race condition in index query (#1356)
+- **M3Coordinator**: Fix to panic responder which was previously not reporting query panics to users (#1353)
```
Comment (benraskin92) [inline CHANGELOG.md]: Also fix the temporal issues (https://github.com/m3db/m3/issues/1344)?

Comment (arnikola) [reply CHANGELOG.md]: That was already fixed in 0.5.0; should we retroactively mention it there instead?

Comment (benraskin92) [reply CHANGELOG.md]: As discussed, was introduced post 0.5.0

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+
+# 0.6.0 (2019-02-19)
+
+## Breaking changes
+
+- **M3Coordinator**: ID generation scheme must now explicitly be added to configs (#1381) [Further information on migrating to 0.6.0](http://m3db.github.io/m3/how_to/query/#migration)
```
Comment (robskillington) [inline CHANGELOG.md]: nit: Can we make this:
```
- **M3Coordinator**: ID generation scheme must be explicitly defined in configs ([Set "legacy" if unsure, further information on migrating to 0.6.0](http://m3db.github.io/m3/how_to/query/#migration)) (#1381) 
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,29 @@
...
+
+## New Features
+
+- **M3DB** (Config): Simplify M3 config options (#1371)
+- **M3Coordinator**: Improve database creation API (#1350)
+- **M3Query**: Add quantile_over_time (#1367) and histogram_quantile (#1372) Prometheus functions
```
Comment (robskillington) [inline CHANGELOG.md]: Can you add the words "Prometheus functions" before the PR number?
e.g.
```
- **M3Query**: Add quantile_over_time (#1367) and histogram_quantile Prometheus functions (#1372) 
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,29 @@
...
+- **Documentation**: Additional documentation for namespace setup and configuration (#1350), etcd (#1354), and M3Coordinator ID generations schemes (#1381, #1385)
```
Comment (robskillington) [inline CHANGELOG.md]: Can you put all PR numbers at the end? e.g. `(#1350, #1381, #1385)`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,29 @@
...
+
+## Performance
+
+- **M3DB** (Index): Add posting list cache (#1370) that should result in a massive improvement for M3DB query performance for most workloads
```
Comment (robskillington) [inline CHANGELOG.md]: Can you add PR number at end? e.g. `most workloads (#1370)`

**Pull request merged**